demo/e2e/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "moduleResolution": "node",
     "outDir": "../dist/out-tsc-e2e",
     "sourceMap": true,
-    "target": "es5",
+    "target": "es6",
```

Comment:
the same, is it needed ?

Reply:
Yes, we need this change for running tests, that use promises